### DISCUSSION TITLE
Is a Version 3.1.5 already planned?

### DISCUSSION THREAD

**Discussion by dr-duplo** [valhalla/valhalla#3594] [Q&A] [answered]
First of all, many thanks for that wonderful piece of software.
I wonder if there's a release plan for version 3.1.5.

**Comment by kevinkreiser** [comment, marked as answer]
sorry i know its been a while. yes we will do a release. i would like to merge a few outstanding things before then though:

https://github.com/valhalla/valhalla/pull/3428
https://github.com/valhalla/valhalla/pull/3532
https://github.com/valhalla/valhalla/pull/3576

**Reply by dr-duplo** [reply]
thanks @kevinkreiser. if help is wanted, please reach out.
